<?xml version="1.0"?>
<launch>

	<!-- Set relevant parameters. -->
	<param name="youBotConfigurationFilePath" type="string" value="$(find youbot_driver)/config"/>

	<!-- Default name values --> 
	<param name="youBotBaseName" type="str" value="youbot-base"/>
	<param name="youBotArmName1" type="str" value="youbot-manipulator"/>
	<param name="driverCycleFrequencyInHz" value="30"/>
	
	<rosparam file="$(find youbot_driver_interface)/param/youbot_configuration.yaml" command="load" />

	<node name="youbot_driver" pkg="youbot_driver_interface" type="youbot_driver_interface" output="screen"/>

	<param name="robot_description" command="$(find xacro)/xacro.py '$(find youbot_description)/robots/youbot.urdf.xacro'"/>

</launch>
